lightweight-term-count-update.php

```diff
@@ -9,7 +9,7 @@
  * Domain Path:     /languages
  * Version:         0.1.0
  *
- * @package 		Lightweight_Term_Count_Update
+ * @package         Lightweight_Term_Count_Update
```

Comment:
Why is this spacing change included as part of this PR?

Reply:
This spacing change is added to pass one of the build jobs due to PHPCS error: https://travis-ci.org/Automattic/lightweight-term-count-update/jobs/376770856#L642. I must have mentioned this on respective commit, my bad.

This particular check must not have been added earlier hence the error popped up now.